examples/client/Locomotion/patches/react-native+0.71.0.patch

```diff
@@ -0,0 +1,201 @@
++
++    spec.subspec 'Hermes' do |ss|
++      ss.source_files = ''
++      ss.public_header_files = 'API/hermes/*.h'
++      ss.header_dir = 'hermes'
++    end
++
++    spec.subspec 'JSI' do |ss|
++      ss.source_files = ''
++      ss.public_header_files = 'API/jsi/jsi/*.h'
++      ss.header_dir = 'jsi'
++    end
++
++    spec.subspec 'Public' do |ss|
++      ss.source_files = ''
++      ss.public_header_files = 'public/hermes/Public/*.h'
++      ss.header_dir = 'hermes/Public'
++    end
++
++    hermesc_path = ""
++
++    if ENV.has_key?('HERMES_OVERRIDE_HERMESC_PATH') && File.exist?(ENV['HERMES_OVERRIDE_HERMESC_PATH']) then
++      hermesc_path = ENV['HERMES_OVERRIDE_HERMESC_PATH']
++    else
++      # Keep hermesc_path synchronized with .gitignore entry.
++      ENV['REACT_NATIVE_PATH'] = react_native_path
++      hermesc_path = "${REACT_NATIVE_PATH}/sdks/hermes-engine/build_host_hermesc"
++      # NOTE: Prepare command is not run  if the pod is not downloaded.
++      spec.prepare_command = ". #{react_native_path}/sdks/hermes-engine/utils/build-hermesc-xcode.sh #{hermesc_path}"
++    end
++
++    spec.user_target_xcconfig = {
++      'FRAMEWORK_SEARCH_PATHS' => '"$(PODS_ROOT)/hermes-engine/destroot/Library/Frameworks/iphoneos" ' +
++                                  '"$(PODS_ROOT)/hermes-engine/destroot/Library/Frameworks/iphonesimulator" ' +
++                                  '"$(PODS_ROOT)/hermes-engine/destroot/Library/Frameworks/macosx" ' +
++                                  '"$(PODS_ROOT)/hermes-engine/destroot/Library/Frameworks/catalyst"',
++      'OTHER_LDFLAGS' => '-framework "hermes"',
++      'HERMES_CLI_PATH' => "#{hermesc_path}/bin/hermesc"
++    }
++
++    spec.script_phases = [
++      {
++        :name => 'Build Hermes',
++        :script => <<-EOS
++        . ${PODS_ROOT}/../.xcode.env
++        export CMAKE_BINARY=${CMAKE_BINARY:-#{%x(command -v cmake | tr -d '\n')}}
++        . ${REACT_NATIVE_PATH}/sdks/hermes-engine/utils/build-hermes-xcode.sh #{version} #{hermesc_path}/ImportHermesc.cmake
++        EOS
++      },
++      {
++        :name => 'Copy Hermes Framework',
++        :script => ". ${REACT_NATIVE_PATH}/sdks/hermes-engine/utils/copy-hermes-xcode.sh"
++      }
++    ]
++  end
++end
+diff --git a/node_modules/react-native/sdks/hermes/hermes-utils.rb b/node_modules/react-native/sdks/hermes/hermes-utils.rb
+new file mode 100644
+index 0000000..0842496
+--- /dev/null
++++ b/node_modules/react-native/sdks/hermes/hermes-utils.rb
+@@ -0,0 +1,26 @@
++# Copyright (c) Meta Platforms, Inc. and affiliates.
++#
++# This source code is licensed under the MIT license found in the
++# LICENSE file in the root directory of this source tree.
++
++require 'net/http'
++require 'rexml/document'
++
++# This function downloads the nightly prebuilt version of Hermes based on the passed version
++# and save it in the node_module/react_native/sdks/downloads folder
++# It then returns the path to the hermes tarball
++#
++# Parameters
```

Comment:
remove anything thats not neccarcy in the patch files: comments, local changes , etc